tests/test_pulse.py

```diff
@@ -363,4 +363,4 @@ def test_pulse_wigner_de_equals_omega(make_pulse, envelope_type):
     for i, tprime in enumerate(tprimes):
         effective_omega = omega + 2 * chirp * tprime
         de = effective_omega
-        assert pulse.wigner_transform(tprime, de) == pytest.approx(s[envelope_type][i], abs=1e-15, rel=1e-15)
```

Comment:
I think unfortunately putting the snapshots inside `pytest.approx` doesn't quite work and these tests were failing for me locally for some reasons, even though they were passing these tolerances.

Reply:
Interesting, I had problems with them too. It never showed me the tolerance when I used the value from the snapshot. I haven't thought about using `approx` on the other value.